awsconfigs/common/aws-authservice/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+)
+
+var port string
+var redirectURL string
+
+func init() {
+	port = os.Getenv("PORT")
+	redirectURL = os.Getenv("LOGOUT_URL")
+
+}
+
+// LogoutHandler expires ALB Cookies and redirects to Cognito Logout Endpoint
+func LogoutHandler(w http.ResponseWriter, r *http.Request) {
+	log.Println("Traffic reached LogoutHandler")
+
+	// There are 4 possible AWSELBAuthSessionCookies
```

Comment:
Is it possible that there might be less than four cookies ?

Reply:
Yes it is possible, in-fact currently there only appears to be two cookies. However setting x amount of cookies to expire even if they don't exist doesn't cause problems.